test/zhr_devs/submissions/commands/solution_submitted_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule ZhrDevs.Submissions.Commands.SolutionSubmittedTest do
+  use ExUnit.Case, async: true
+
+  import Commanded.Assertions.EventAssertions
+
+  import ZhrDevs.Fixtures
+
+  alias Commanded.Aggregates.Aggregate
+
+  alias ZhrDevs.App
+
+  alias ZhrDevs.Submissions.{Aggregates, Commands, Events}
+
+  alias ZhrDevs.Submissions.SubmissionIdentity
+
+  describe "SolutionSubmitted command" do
+    setup do
+      valid_opts = fn identity ->
+        [
+          hashed_identity: DomaOAuth.hash(identity),
+          task_uuid: "Jaju6yAv1oZ23NcjJk-1JkOrxCemsH_K-9iRRw0sYRg=",
+          technology: "elixir",
+          solution_path: "test/support/testfile.txt"
```

Comment:
Solution path will be a directory or a zip file (depending on what are you going for here).